docker-compose-generator/docker-fragments/bitcoingold.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  bgoldd:
```

Comment:
why two `d`?

Reply:
No particular reason, the bitcoind is called bgoldd, so I kept it here for consistency.